scene/main/viewport.cpp

```diff
@@ -1700,7 +1700,8 @@ void Viewport::_gui_input_event(Ref<InputEvent> p_event) {
 
 				if (gui.tooltip_popup) {
 					if (gui.tooltip_control) {
-						String tooltip = _gui_get_tooltip(over, gui.tooltip_control->get_global_transform().xform_inv(mpos));
+						Control *tooltip_owner = nullptr;
+						String tooltip = _gui_get_tooltip(over, gui.tooltip_control->get_global_transform().xform_inv(mpos), &tooltip_owner);
```

Comment:
Was this intentional? If we don't need the `tooltip_owner` here we likely don't need to save it.

Reply:
Oh, right, forgot to remove from testing.